address_book/api/api.proto

```diff
@@ -5,10 +5,30 @@ option go_package = "address_book/internal/pb";
 import "google/api/annotations.proto";
 
 service AddressBookService {
-  rpc AddContact(AddContactRequest) returns (AddContactResponse) {}
-  rpc FindContact(FindContactRequest) returns (FindContactResponse) {}
-  rpc DeleteContact(DeleteContactRequest) returns (DeleteContactResponse) {}
-  rpc UpdateContact(UpdateContactRequest) returns (UpdateContactResponse){}
+  rpc AddContact(AddContactRequest) returns (AddContactResponse) {
+    option (google.api.http) = {
+      post: "/addContact"
```

Comment:
I would suggest to keep naming to verbs only.

Reply:
Done